backend/db/migrate/20250807201734_add_internal_document_fields.rb

```
@@ -0,0 +1,14 @@
+class AddInternalDocumentFields < ActiveRecord::Migration[8.0]
+  def change
+    add_column :documents, :file_storage_key, :string
+    add_column :documents, :signature_data, :jsonb
+    add_column :documents, :status, :string, default: 'unsigned'
```

RuboCop (1.76.1)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)